pkg/registry/pod/rest/log.go

```diff
@@ -58,6 +58,9 @@ func (r *LogREST) Get(ctx api.Context, name string, opts runtime.Object) (runtim
 	if err != nil {
 		return nil, err
 	}
+	if location.Host == "" {
```

Comment:
shouldn't need to add these checks here... the location streamer should handle that, right?